docs/DeveloperGuide.md

```diff
@@ -295,6 +295,8 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 | `* *`    | user                    | view contacts related to my search query                   | find contacts even when I mistype their name                               |
 | `* *`    | user                    | hide private contact details                               | minimize chance of someone else seeing them by accident                    |
 | `* *`    | user                    | show private contact details                               | view them when I need to                                                   |
+| `* *`    | user                    | have an autocomplete for the names that I am searching     | search faster by names and minimize the chance of an unsuccessful search.  |
+| `* *`    | user                    | have an autocomplete for the tags that I am searching      | search faster by tags and minimize the chance of an unsuccessful search.   |
```

Comment:
@Eugene-Ong-W-X I saw that you have added this user story - `As a user, I want to view suggestions of contacts matching the name I am typing so that I can find a contact without typing the full name.`

I am not sure if the user story you have written means the user can search by partial names, i.e when a user enters `search n/An`, SoConnect will show a list of contacts with names related to `An` or do you mean the autocomplete feature.

Let me know if the two stories I have added are the same as what you have written.

Reply:
Okay, I have updated the user story.